### PR TITLE
filter works with boolean values

### LINKED ISSUE #57: Querying on boolean attributes doesn't work when they are false.
The LS Adapter doesn't support querying on boolean model attributes when the values are false:

From [adapter.js:217](https://github.com/funkensturm/ember-local-storage/blob/master/addon/adapters/adapter.js#L217)
```js
        // Attributes
        if (key === 'id' || key === 'type') {
          recordValue = data[key];
        } else {
          key = serializer.keyForAttribute(key);
          recordValue = data.attributes ? data.attributes[key] : null;
        }

        if (recordValue) { // <----------------- this is the problem line
          return this._matches(recordValue, queryValue);
        }
```

If the model has an attribute  like `isAwesome: DS.attr('boolean', {defaultValue: false}),`

Then a query like `store.query('my-model', { filter: {isAwesome: false } })` returns 0 results.

### PR DESCRIPTION
**Pull request by bekzod**
fixes #57

### CODE REVIEW

**Comment by fsmanuel** [comment]
@bekzod thanks! can you add a test? If you need help with the test let me know.

**Comment by bekzod** [comment]
ok will add test

File: addon/adapters/adapter.js [hunk 1]
```diff
@@ -129,14 +131,14 @@ export default Adapter.extend(ImportExportMixin, {
       console.log(url, type, options);
     }
 
-    return new Ember.RSVP.Promise((resolve, reject) => {
-      if (this[`_handle${type}Request`]) {
-        const data = this[`_handle${type}Request`](url, options.data);
-
+    return new RSVP.Promise((resolve, reject) => {
+      let handler = this[`_handle${type}Request`];
```
Comment (fsmanuel) [inline addon/adapters/adapter.js]: `const`

**Comment by bekzod** [comment]
@fsmanuel added tests, let me know if there is something wrong

**Comment by fsmanuel** [comment]
@bekzod thanks!

**Pull request merged**